server/gokbg3/grails-app/services/org/gokb/FTUpdateService.groovy

```diff
@@ -389,6 +398,76 @@ class FTUpdateService {
         }
 
         result.componentType = kbc.class.simpleName
+        result.tippTitleMedium = kbc.title ? kbc.title.medium : ""
+
+        result.accessStartDate = kbc.accessStartDate ? kbc.accessStartDate.toString() : ""
+        result.accessEndDate = kbc.accessEndDate ? kbc.accessEndDate.toString() : ""
```

Comment:
We are using a centralized DateFormatService to format Date objects. In this case `dateFormatService.formatIsoTimestamp(Date date)` would be the way to go.